src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';');
+  const result = {};
+
+  sourceArray.forEach((elem) => {
+    const newElem = elem.trim();
+    const arr = newElem.split(':');
```

Comment:
[CODE STYLE] - The variable name `arr` is not descriptive and decreases code readability. Consider renaming it to something more meaningful, such as `declarationParts`.